### PR TITLE
disable speed perturbation by default

### PR DESCRIPTION
**Pull request by JinZr**

### CODE REVIEW

**Comment by danpovey** [comment]
We should probably do this, even though it may make some of the recipes not fully reproducible (i.e. it would require to change the learning rate schedules and num-epochs).
It might make sense to actually change the LR schedules and epochs at the same time.  This would not be fully
foolproof, since some people will have previously-dumped non-perturbed features on disk.  But it will ensure that
from a fresh checkout things would be basically reproducible.
(if README files have example usages, we could perhaps just write the alternative command lines that you would use
if the features were not dumped with speed perturbation, with the num-epochs and if applicable lr-epochs multiplied by 3.)

**Comment by pzelasko** [comment]
It doesn’t help anymore?

**Comment by JinZr** [comment]
Not really. The thing is when the amount of data reaches a certain point,
speed perturbation stops bringing performance improvement to the system and
would be a waste of space and time. So we figure that we can leave a switch
and let users decide whether they want it or not.

**Comment by pkufool** [comment]
> It doesn’t help anymore?

We did experiments on librispeech dataset, the performance of  speed-pertur is the same as no speed-pertur (3 times epochs).

**Comment by yfyeung** [comment]
> We did experiments on librispeech dataset, the performance of speed-perturb is the same as no speed-perturb (3 times epochs).

Somehow slightly worse than with perturb

**Review by yfyeung — APPROVED** [review]

**Review by yfyeung — CHANGES_REQUESTED** [review]
Maybe you can use:

if speed_perturb and "train" in partition:

**Comment by JinZr** [comment]
applied requested changes to the latest commit

**Comment by danpovey** [comment]
There is already a PR (sorry IDK the PR#) to change the arg to false in the
prepare.sh scripts.

File: egs/aidatatang_200zh/ASR/local/compute_fbank_aidatatang_200zh.py [hunk 1]
```diff
@@ -109,7 +109,12 @@ def get_args():
         default=80,
         help="""The number of mel bins for Fbank""",
     )
-
+    parser.add_argument(
+        "--speed-perturb",
+        type=bool,
```
Comment (csukuangfj) [inline egs/aidatatang_200zh/ASR/local/compute_fbank_aidatatang_200zh.py:114]: Please follow
https://github.com/k2-fsa/icefall/blob/00256a766921dd34a267012b0e2b8ff7d538f0e6/egs/librispeech/ASR/tdnn_lstm_ctc/asr_datamodule.py#L87
to use `str2bool`.

**Review by csukuangfj — COMMENTED** [review]
Thanks! Left a minor comment. Otherwise, it looks good to me.

File: egs/aidatatang_200zh/ASR/local/compute_fbank_aidatatang_200zh.py (hunk 1 above)
Comment (JinZr) [reply egs/aidatatang_200zh/ASR/local/compute_fbank_aidatatang_200zh.py]: thx! fixed this in the latest commit.

**Review by csukuangfj — APPROVED** [review]
Thanks!
